<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" multifile-model="true">
    <data-source source="LOCAL" name="Django default driver - identifier.sqlite" uuid="47a347a6-0b69-4e31-ade9-21a8522ac26c">
      <driver-ref>54ff4025-1743-48c9-a9eb-7003d27c5fbf</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:identifier.sqlite</jdbc-url>
    </data-source>
    <data-source source="LOCAL" name="Sqlite (Xerial) - django.sqlite3" uuid="23d8ad76-654f-44c7-8b58-6770dca36bbc">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:$PROJECT_DIR$/django.sqlite3</jdbc-url>
    </data-source>
  </component>
</project>